documentation/API_documentation/device-swap-API-Readiness-Checklist.md

```diff
@@ -0,0 +1,29 @@
+# API Readiness Checklist
+
+Checklist for device-swap 0.2.0-rc.1 in r2.1
+
+| Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Comments |
```

Comment:
For Spring25 the template was slightly updated, e.g. the instructions and wiki link, but also the last column and its expected content:
```suggestion
| Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Reference information  |
```